cmake/Macros.cmake

```diff
@@ -35,6 +35,17 @@ function(sfml_set_stdlib target)
             message(FATAL_ERROR "Clang is the only supported compiler on macOS")
         endif()
     endif()
+
+    # on some platforms (e.g. ARM), GCC requires linking libatomic to use <atomic> features
+    if(SFML_COMPILER_GCC)
```

Comment:
Does this same problem apply to those using Clang on Raspberry Pis?

Reply:
I've only seen people reporting about this using GCC since it was an active decision of theirs to split the atomic implementation out of the standard runtime library that is automatically linked in. The fact that this doesn't happen when building with GCC for aarch64 (RPi 64-bit) makes me think they regret this decision and have just followed what all other sane compilers do.